src/amo/components/Routes/index.js

```diff
@@ -147,6 +147,12 @@ const Routes = ({ _config = config }: Props = {}) => (
       component={CategoriesPage}
     />
 
+    <Route
+      exact
+      path="/:lang/:application/category/"
```

Comment:
We probably want per-category URLs so it should be something like this:

```
path="/:lang/:application/:visibleAddonType(extensions|themes)/:categorySlug/"
```

Reply:
Why do we need those? I thought this was just about the ability to crawl. I can change it, but this is the most expedient way to accomplish this.

Reply:
Pretty urls are a bit better in terms of crawling experience and we're going to enhance these pages in the future. I think it's best to plan for the future, it's not a lot of work.

We can mimic what is done in `SearchPage`. We don't need to worry about the server redirects, etc. so the `CategoryPage` should be a simple component (famous last words?)